core/workspace_svg.js

```diff
@@ -1226,6 +1228,54 @@ Blockly.WorkspaceSvg.prototype.showContextMenu_ = function(e) {
   redoOption.callback = this.undo.bind(this, true);
   menuOptions.push(redoOption);
 
+  // Option to add a workspace comment.
+  if (this.options.comments) {
+    /**
+     * Option to add a workspace comment..
+     * @private
+     */
+    var addWsComment = function() {
```

Comment:
In a followup, let's pull this into a helper function.  For now just add a TODO.

Reply:
Added TODO